spec/features/team/reset_2fa_spec.rb

```diff
@@ -0,0 +1,133 @@
+describe "Reset two factor authentication", type: :feature do
+  let(:organisation) { create(:organisation) }
+  let(:org_admin_user_1) { create(:user, organisations: [organisation]) }
+  let(:org_admin_user_2) { create(:user, organisations: [organisation]) }
+  let(:org_admin_user_3) { create(:user, organisations: [organisation]) }
+
+  let(:super_admin_organisation) { create(:organisation, super_admin: true) }
```

Comment:
`super_admin` is a flag on the user, not the org. You can probably remove this `super_admin_organisation` as it's not always used anymore.